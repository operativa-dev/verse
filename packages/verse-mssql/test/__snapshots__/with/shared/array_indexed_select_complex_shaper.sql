-- Executing SQL: Parameters: []
select json_query(json_query("t2"."c0", '$[0]'), '$[1]') as "c1", json_query("t2"."c0", '$[1]') as "c2", json_query("t2"."c0", '$[2]') as "c3"
from (
   select '[' + (string_agg(json_array("t1"."ArtistId", "t1"."Name"), ',') + ']') as "c0"
   from "Artist" as "t1"
) as "t2"

