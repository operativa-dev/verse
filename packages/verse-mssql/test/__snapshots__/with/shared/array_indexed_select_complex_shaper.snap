[
  [
    undefined,
    Artist {
      "albums": [],
      "artistId": 2,
      "name": "Accept",
    },
    {
      "foo": "[3,"Aerosmith"]",
    },
  ],
]